.github/workflows/go.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
       - name: Ensure that all files are properly formatted
         run: |
-          FILES=$(go run golang.org/x/tools/cmd/goimports -w -l .)
+          FILES=$(make fmt-fix)
```

Comment:
I think you had `run: make fmt-check` here before and it was very neat looking - did it not work as you wanted? Or did I make that up?

Reply:
I had it this way at some point. But this way we pretty print the offending files